src/main/java/com/coreos/jetcd/EtcdUtil.java

```diff
@@ -0,0 +1,80 @@
+package com.coreos.jetcd;
+
+import com.coreos.jetcd.api.Event;
+import com.coreos.jetcd.api.ResponseHeader;
+import com.coreos.jetcd.data.ByteSequence;
+import com.coreos.jetcd.data.EtcdHeader;
+import com.coreos.jetcd.data.KeyValue;
+import com.coreos.jetcd.watch.WatchEvent;
+import com.google.protobuf.ByteString;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This util is to convert api class to client class.
+ */
+public class EtcdUtil {
```

Comment:
How about making EtcdUtil package private? I think this is only used in jetcd internal.